### ISSUE TITLE
Duplicate Menu Items in Bundle Menu - Essentially Cosmetic

### ISSUE THREAD

**Issue by tedmasterweb** [sequelpro/sequelpro#2129]
Menu items in the Bundles menu are duplicated.

Preconditions: none

Steps to Reproduce
1. Launch Sequel Pro
2. Click Bundles menu
3. Navigate to Open (for example)

Expected Result:
A single menu item called "Open selected table with…"

Actual Result:
"Open selected table with…" is shown twice.

Additional info:
This seems to be the case with all the actual commands in the Bundle menu. Here are two screen shots that show the behavior:
- http://grab.by/HSeQ
- http://grab.by/HSf0

I have the prior production release still in the Applications folder and thus, this new version is named Sequel Pro 2.app - not sure if that matters.

Sequel Pro Version: 1.1 (Build 4468 (15800a8))
Mac OS X version: 10.8.5

**Comment by abhibeckert** [comment]
I can't reproduce this bug. Can anyone else?

**Comment by tedmasterweb** [comment]
This morning I opened my original version of Sequel Pro (1.0.2 Build 4096) and did not see the behavior. Then I re-opened 1.1 and was also unable to reproduce it. Are there any local files that might get modified when 1.1 is opened that I can reset to their original state to try again? Like I said in the title, I think this is mostly a cosmetic issue, so, low priority, especially since neither of us is unable to reproduce.

**Comment by stevegk** [comment]
I have seen this immediately after updating. Closing and reopening the app or rebuilding the bundles fixes it.

**Comment by abhibeckert** [comment]
Perhaps delete these two files:

 * `~/Library/Application Support/Sequel Pro`
 * `~/Library/Preferences/com.sequelpro.SequelPro.plist`

Then run Sequel Pro 1.0.2, then close it and run Sequel Pro 1.1RC1. I don't have time to test that right now.

**Comment by tedmasterweb** [comment]
I can confirm that following the recommended procedure the behavior has
returned. Quitting the app and reopening it makes the behavior disappear.
HTH.

Ted

**Comment by rowanbeentje** [comment]
Thanks Ted!

So presumably as we "upgrade" bundles we're not cleaning up the duplicate entries in the menus.

**Comment by dmoagx** [comment]
Can confirm.
* Delete Prefs and App Data
* Open 1.0.2 and Quit
* Open 1.1rc1 -> Duplicate Items

**Comment by tedmasterweb** [comment]
Do you need anything else from me at this time?

**Comment by abhibeckert** [comment]
@tedmasterweb all good, we don't need anything else and will get it fixed before we ship version 1.1.

**Comment by tedmasterweb** [comment]
Thrilled I could help. If you need a hand with anything else (not
programming, I'm not terribly good at Objective-C, or even C++ for that
matter), please let me know. Great product!

Ted

**Comment by abhibeckert** [comment]
@tedmasterweb yes there are other things you could help with, can you send me an email? abhi@abhibeckert.com
